hls-graph/src/Development/IDE/Graph/Internal/Types.hs

```diff
@@ -187,7 +187,6 @@ instance NFData RunMode where rnf x = x `seq` ()
 -- | How the output of a rule has changed.
 data RunChanged
     = ChangedNothing -- ^ Nothing has changed.
-    | ChangedStore -- ^ The stored value has changed, but in a way that should be considered identical (used rarely).
```

Comment:
I do not think it is ever used for would be used, we could added back if there is a need for this.